<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Aircondition</class>
 <widget class="QWidget" name="Aircondition">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="current_onoff">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="current_mode">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="current_temp">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="current_wind">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="on_off">
        <property name="text">
         <string>ON/OFF</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mode">
        <property name="text">
         <string>SetMode</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="temperature">
        <property name="text">
         <string>SetTemperature</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="wind">
        <property name="text">
         <string>SetWind</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="warn">
        <property name="text">
         <string>Warning Notice</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>50</number>
      </property>
      <property name="leftMargin">
       <number>50</number>
      </property>
      <property name="rightMargin">
       <number>50</number>
      </property>
      <item>
       <widget class="QPushButton" name="quit">
        <property name="text">
         <string>Quit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="back">
        <property name="text">
         <string>Return</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
